### ISSUE TITLE
yafray -> yafaray

### ISSUE THREAD

**Issue by glixx** [K-3D/k3d#15]
yafray was renamed with name yafaray.
for executable:
s/yafray/yafaray-xml/

for strings in program:
s/yafray/yafaray/
s/Yafray/Yafaray/

**Comment by barche** [comment]
Yes, sorry for the long wait. Seems I missed this original issue. I'm in the middle of the GTK3 conversion, so I'll do this afterwards. Note that my time for K-3D is very limited, so I work on it very sporadically.
